api/src/main/java/org/hawkular/alerts/api/model/condition/Condition.java

```diff
@@ -86,38 +85,35 @@
      * Number of conditions associated with a particular trigger.
      * i.e. 2 [ conditions ]
      */
-    @ApiModelProperty(value = "Number of conditions associated with a particular trigger. This is a read-only value " +
+    @DocModelProperty(description = "Number of conditions associated with a particular trigger. This is a read-only description " +
```

Comment:
In the text: description -> value

Reply:
Oh, good catch.
My IDE did an extra refactoring and it seems it changed some words inside text strings.
I thought I fixed them. 
Going to fix them, I guess following issues are related to this problem.